dev/source/docs/mavlink-get-set-home-and-origin.rst

```diff
@@ -166,7 +243,6 @@ The message definition can be found `here <https://mavlink.io/en/messages/common
 
 The example command below can be copy-pasted into MAVProxy (aka SITL) to test this command.  Before running these commands enter, "module load message"
 
-- module load message
```

Comment:
Any reason to remove this here rather than in the preceding sentence?

Reply:
ok, removed the upper message instead.